### DISCUSSION TITLE
Can this module download daily / hourly history for IG Client Sentiment?

### DISCUSSION THREAD

**Discussion by DROBNJAK** [ig-python/ig-markets-api-python-library#336] [Q&A]
It would be interesting to do some backtests with **IG Client Sentiment** as an input, so question is can this lib pull of Client Sentiment, and if it can how far back in days or in hours?

**Comment by bug-or-feature** [comment]
That's the wrong question. What is important is what the IG API allows - [see their docs](https://labs.ig.com/rest-trading-api-reference.html). All this lib does is provide a lightweight Python wrapper for their APIs.

As you can see there's no history function for client sentiment, so you would have to create your own history. And that is not very practical due to the [rate limits](https://labs.ig.com/faq.html#limits)
